nautobot_secrets_providers/providers/one_password.py

```diff
@@ -0,0 +1,110 @@
+"""1Password Secrets Provider for Nautobot."""
+
+import asyncio
+
+from django import forms
+from django.conf import settings
+from nautobot.core.forms import BootstrapMixin
+from nautobot.extras.secrets import SecretsProvider, exceptions
+
+try:
+    from onepassword.client import Client
+except ImportError:
+    Client = None
+
+from nautobot_secrets_providers import __version__
+
+__all__ = ("OnePasswordSecretsProvider",)
+
+
+async def get_secret_from_vault(vault, item, field, token, section=None):
+    """Get a secret from a 1Password vault.
+
+    Args:
+        vault (str): 1Password Vault where the secret is located.
+        item (str): 1Password Item where the secret is located.
+        field (str): 1Password secret field name.
```

Comment:
Any reasonable default for this arg?

Reply:
Probably password, but it could be any field in 1Password of the supported field types (and the field name doesn't have to match the field type.